src/tools.cpp

```diff
@@ -638,6 +638,9 @@ MagicEffectNames magicEffectNames = {
 	{"orangeteleport",	CONST_ME_ORANGETELEPORT},
 	{"greyteleport",	CONST_ME_GREYTELEPORT},
 	{"lightblueteleport",	CONST_ME_LIGHTBLUETELEPORT},
+	{"fatal", CONST_ME_FATAL},
+	{"dodge", CONST_ME_DODGE},
+	{"hourglass", CONST_ME_HOURGLASS}
```

Comment:
formatting not following pattern above

Reply:
the formatting appears ok in vscode, but I'm not so sure if it's correct

Reply:
I think it refers to this:
https://github.com/otland/forgottenserver/blob/8ca885f7fd38de056072a818e116d791eb85165c/src/tools.cpp#L640

why do you remove the `,`

Reply:
> I think it refers to this:
> 
> https://github.com/otland/forgottenserver/blob/8ca885f7fd38de056072a818e116d791eb85165c/src/tools.cpp#L640
> 
> why do you remove the `,`
> 
> ![image](https://user-images.githubusercontent.com/28090948/145456054-53dfd0c6-f23e-4570-a398-bdfad0e055c5.png)

corrected, thanks!